lib/src/web/web_sqlite_open_factory.dart

```diff
@@ -1,17 +1,28 @@
 import 'dart:async';
 
-import 'package:drift/wasm.dart';
 import 'package:sqlite3/wasm.dart';
+import 'package:sqlite3_web/sqlite3_web.dart';
 import 'package:sqlite_async/sqlite_async.dart';
-import 'package:sqlite_async/src/web/database/connection/drift_sqlite_connection.dart';
 import 'package:sqlite_async/src/web/web_mutex.dart';
 
+import 'database.dart';
+
 /// Web implementation of [AbstractDefaultSqliteOpenFactory]
 class DefaultSqliteOpenFactory
     extends AbstractDefaultSqliteOpenFactory<CommonDatabase> {
+  // todo: For users with multiple databases, the WebSqlite should be shared
```

Comment:
A single `WebSqlite` instance can handle multiple databases, but every `WebSqlite` instance potentially spawns its own worker so having more than one should be avoided.

If I remember correctly, we're currently using a new `DefaultSqliteOpenFactory` for each database (and it has to be like that because the path is bound to the factory). So perhaps we could have a static `Map<SqliteOptions, Future<WebSqlite>>` around that lazily creates a new `WebSqlite` instance only if a different URI for the wasm module or the worker is used. Otherwise the existing instance can be re-used.

Reply:
That makes sense. We can do that on our end.